rollup.config.mjs

```diff
@@ -14,19 +14,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { computeBanner } from './scripts/shared/banner.mjs';
 // generate warning when running with Node 18
 // (node:75278) ExperimentalWarning: Importing JSON modules is an experimental feature. This feature could change at any time
```

Comment:
**nitpick:** This comment must be moved with "import pkg..." as it relates to it